src/rgb/carbonado.rs

```diff
@@ -1,10 +1,13 @@
 use amplify::confinement::{Confined, U32};
 use anyhow::Result;
 use rgbstd::persistence::Stock;
+use serde_json::to_vec;
 use strict_encoding::{StrictDeserialize, StrictSerialize};
 
 use crate::carbonado::{retrieve, store};
 
+use super::structs::RgbAccount;
```

Comment:
Let's pull this from `crate::`
Also, please check other places the super pattern has been used and see if we can't reduce the different patterns that are needed for consistency's sake.